frontends/web/package.json

```diff
@@ -20,7 +20,8 @@
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "react-i18next": "11.16.8",
-    "react-router-dom": "6.3.0"
+    "react-router-dom": "6.3.0",
+    "request-address": "^0.0.3"
```

Comment:
Minor detail: for non-devdeps I would prefer to lock it in the end, but feel free to keep the semver range in staging.

To add a locked npm version do `npm install --save-exact request-address` or preferably `npm install --save-exact --ignore-scripts request-address`. The ignore-scripts flags does not execute postinstall hooks, there is almost never a reason to execute a script when installing npm deps.

Reply:
Done :+1: